ui-participant/src/hub/OutreachTasks.tsx

```diff
@@ -62,8 +62,11 @@ export default function OutreachTasks({ enrollees, studies }: {enrollees: Enroll
   }
 
   useEffect(() => {
-    loadOutreachActivities()
-  }, [enrollees])
+    if (enrollees.length) {
```

Comment:
Nit: I am unfamiliar with this syntax so my instinct would be to make it clearer (`enrollees.length > 0`)

Reply:
I think it's a shorthand common in JS and Ruby, relying on the fact that 0 evaluates to `falsy`. I like it because once you get used to it, it's easier to read at a glance.